de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/trx/api/ITrxManager.java

```diff
@@ -320,6 +320,9 @@ default void runInThreadInheritedTrx(final Runnable runnable)
 	 */
 	void runOutOfTransaction(TrxRunnable r);
 
+	/** Run after current transaction is committed. If no transaction, the code is executed right away. */
```

Comment:
```suggestion
	/** Run after the next time the current transaction is committed (i.e. run just once). If there is no current transaction, the code is executed right away. 
	 * This is a convenience method that utilizes the TrxListenerManager.
	 */
```

Reply:
Pls don't just resolve (discard) my comments without further writing why you think it's not a good idea.
In this case, i think the extended javadoc i proposed offers some fud reducing information for users of the API..

EDIT: nvm, i assume it slipped through, plus it's not really that important to have a more verbose javadoc there